distribution/docs/src/main/resources/content/_architectures/_catalogFrameworks/standard-catalog-framework.adoc

```diff
@@ -31,8 +31,6 @@ If no local catalog provider is configured, the site information returned includ
 
 The Standard Catalog Framework is bundled as the `catalog-core-standardframework` feature and can be installed and uninstalled using the normal processes described in Configuration.
 
-When this feature is installed, the Catalog Fanout Framework App feature `catalog-core-fanoutframework` should be uninstalled, as both catalog frameworks should not be installed simultaneously.
```

Comment:
❓ This seems a little out of place for this PR... is this meant to be here?

Reply:
I just realized this is a port PR, so it's cool if this stays, just doesn't seem to fit under the commit message.

Reply:
No, it's not directly related. It came up for someone else and I just deleted it here instead of making them open a separate PR. I squashed the commits on the other PR